atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/expectations/ExpectationsConfigurations.java

```diff
@@ -14,24 +14,11 @@
  * limitations under the License.
  */
 
-package com.palantir.atlasdb.transaction.impl.expectations;
+package com.palantir.atlasdb.transaction.api.expectations;
 
-import org.derive4j.Data;
+public final class ExpectationsConfigurations {
```

Comment:
Does this need to be its own class? Could we not have a static constuctor on ExpectationsConfig for this - see CassandraRuntimeConfig for example